exercises/practice/decimal/.meta/Cargo-example.toml

```diff
@@ -4,5 +4,5 @@ name = "decimal"
 version = "0.1.0"
 
 [dependencies]
-num-bigint = "0.1.40"
-num-traits = "0.1.40"
+num-bigint = "0.4.4"
+num-traits = "0.2.16"
```

Comment:
Got some compiler warnings that these old crates contained code which only compiled due to an unsoundness in rustc. They will stop compiling in the future.